### PR TITLE
*P1 - FE/S - Account dashboard [16-20] [reserved]

### PR DESCRIPTION
**Pull request by Space-zee**

### CODE REVIEW

File: frontend/app/features/account/controllers/AccountController/index.tsx [hunk 1]
```diff
@@ -30,48 +39,46 @@ export const AccountController = () => {
...
-                currentValue={reward.computed.currentValue}
-                currentRentBalance={reward.rewardAmountDe6}
-                // totalRentEarned={-1}
-              />
-            );
-          })}
+    <>
+      <div className={s.root}>
+        <SideMenu className={s.pageMenu}>
+          <SideMenuNavItem icon={<IconProperty/>} title="Asset Overview" url="/account-v2"/>
```
Comment (ruslanpalagin) [inline frontend/app/features/account/controllers/AccountController/index.tsx:45]: please use paths.listAssets() to get route

File: frontend/app/features/account/views/DetailsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      <tr className={s.trFirst}>
+        <td className={clsx(s.accountDetails, s.horizontalBorder)}>
+          <div>
+            <div className={s.iconWrapper}>
+              <Image src={walletIcon} alt={'walletIcon'}/>
+            </div>
+          </div>
+          <div>
+            <p className={s.details_text_grey}>Current Account Value</p>
+            <p className={s.details_text_purple}>$1 380.37</p>
```
Comment (ruslanpalagin) [inline frontend/app/features/account/views/DetailsTable.tsx:28]: Probably should be a real value.

File: frontend/app/features/assets/controllers/AdminAssetsListController.tsx [hunk 3]
```diff
@@ -18,19 +18,18 @@ export const AdminAssetsListController = () => {
...
         <div className={s.contentWrapper}>
           <SideMenu className={s.pageMenu}>
-            <SideMenuNavItem icon={<IconProperty />} title="Asset Overview" url="/account-v2" />
-            <SideMenuNavItem icon={<IconCoins />} title="Marketplace" url="/assets-v2" />
-            <SideMenuNavItem icon={<IconGift />} title="Refer and Earn" url="#" />
-            <SideMenuNavItem icon={<IconHistory />} title="Transaction" url="#" />
-            <SideMenuDivider />
-            <SideMenuNavItem icon={<IconSettings />} title="Settings" url="#" />
-            <SideMenuNavItem icon={<IconNotification />} title="Notifications" url="#" />
+            <SideMenuNavItem icon={<IconProperty/>} title="Asset Overview" url="/account-v2"/>
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/controllers/AdminAssetsListController.tsx:26]: please use paths.ROUTENAME()

File: frontend/app/features/assets/views/SideMenu/icons/IconCoins.tsx [hunk 4]
```diff
@@ -10,58 +10,58 @@ const IconCoins = ({ width = 24, height = 24 }) => (
     <path
       opacity="0.2"
       d="M9 11.25C13.1421 11.25 16.5 9.73896 16.5 7.875C16.5 6.01104 13.1421 4.5 9 4.5C4.85786 4.5 1.5 6.01104 1.5 7.875C1.5 9.73896 4.85786 11.25 9 11.25Z"
-      fill="#58667E"
+      fill={color}
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/views/SideMenu/icons/IconCoins.tsx:13]: please use CSS in order to style

File: frontend/app/shared/walletsConnect/chainInfo.ts [hunk 5]
```diff
@@ -98,8 +98,7 @@ export const getActiveConfig = () :ChainInfo | null => {
     const configNameByDomain =
       {
         'localhost': CHAIN_INFO[SupportedChainId.ARBITRUM_GOERLI],
-        'dev.binaryx.com': CHAIN_INFO[SupportedChainId.ARBITRUM_GOERLI],
-        'staging.binaryx.com': CHAIN_INFO[SupportedChainId.ARBITRUM_GOERLI],
+        'i2.binaryx.com': CHAIN_INFO[SupportedChainId.ARBITRUM_GOERLI],
```
Comment (ruslanpalagin) [inline frontend/app/shared/walletsConnect/chainInfo.ts:101]: why?

'dev.binaryx.com': CHAIN_INFO[SupportedChainId.ARBITRUM_GOERLI],
        'staging.binaryx.com': CHAIN_INFO[SupportedChainId.ARBITRUM_GOERLI],
        
this is correct

**Review by ruslanpalagin — CHANGES_REQUESTED** [review]

**Comment by ruslanpalagin** [comment]
please keep the naming for files:
public/(shared|feature)/MODULE_NAME/FILENAME
example:
frontend/public/svg/accountOverview/graphic.svg -> frontend/public/feature/account/graphic.svg

**Review by ruslanpalagin — CHANGES_REQUESTED** [review]

File: contracts_arbitrum/scripts/mintUsdtf.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import {Contract, ethers, Wallet} from "ethers";
+const usdtfABI  = require('../artifacts/contracts/UsdtfToken.sol/UsdtfToken.json')
```
[the remark this answers is not in the document]
Comment (Space-zee) [reply contracts_arbitrum/scripts/mintUsdtf.ts:2]: +

File: frontend/app/features/account/controllers/AccountController/index.tsx [hunk 7]
```diff
@@ -1,20 +1,29 @@
+//@ts-nocheck
```
[the remark this answers is not in the document]
Comment (Space-zee) [reply frontend/app/features/account/controllers/AccountController/index.tsx:1]: +

File: frontend/app/features/assets/controllers/AdminAssetsListController.tsx (hunk 3 above)
Comment (Space-zee) [reply frontend/app/features/assets/controllers/AdminAssetsListController.tsx:26]: +

File: frontend/app/shared/walletsConnect/chainInfo.ts (hunk 5 above)
Comment (Space-zee) [reply frontend/app/shared/walletsConnect/chainInfo.ts:101]: +

**Comment by Space-zee** [comment]
https://app.clickup.com/t/33yzpzf

**Review by ruslanpalagin — APPROVED** [review]

**Comment by ruslanpalagin** [comment]
сабтаск https://app.clickup.com/t/3cxgk1t

**Pull request merged**